.env.prod

```diff
@@ -0,0 +1,10 @@
+DATA_FOLDER=data-dev
```

Comment:
ah sorry, the comments above apply to the env.prod environment.  So I think that in the prod environment we shall define only what the Differ service needs, i.e. the API port, and nothing else, whereas the dev environment needs also a Fuseki service for testing. 

You may also argue that we shall deliver Rdf differ as a compound of two services Rdf-differ + Fuseki into prod as well. This might not be a bad idea but has to be handled with care and **very** very well documented.